.github/workflows/workflow.yaml

```diff
@@ -213,3 +213,22 @@ jobs:
         name: generatorexpansion-logs
         path: |
           logs-*.json
+  
+  helm_lint_test:
```

Comment:
can we move this to a new workflow so it only runs if manifest_staging/charts is altered